content/post/the-future-is-rusty.en.md

```diff
@@ -0,0 +1,69 @@
++++
+title = 'The Future is Rust(y)?'
+slug = 'post'
+image = 'https://cdn.dribbble.com/users/241205/screenshots/4977779/drib_00_webasm_tank.png'
+date = "2019-06-01T00:00:00"
+author = 'Swarnim Arun'
+tags=['rust','prgramming']
+categories =['code-philosophy']
+description = 'My quick take on how I see Rust as the new leader of this ever changing programming world'
++++
+
+<b>Rust is the system programming language for the future.</b> Quite a presumptuous statement, said my confreres. I would be inclined to agree with them but for the experience of the agony of using C.
+
+![](https://doc.rust-lang.org/nomicon/img/safeandunsafe.svg)
+
+> “Having been tortured by C during my years of wiring security-critical software, I don’t think I exaggerate when I compare programming in it with walking through a minefield.”
+by Ivan Ristic, Founder of SSL Labs
+
+I will not add more comments to the above statement as I believe it to be the most accurate description of C language by far. I won't proclaim myself as a C language expert nevertheless I assume all the 'C citizens' here would agree with it.
+
+
+### But sire isn't C++ the next step? Why Rust if we have C++?
+C++, as described by Linus Torvalds, it's a not-so-good programming language(actually he called it horrible and crappy). Or as stated by Donald Knuth, "C++ has a lot of good features, but it has a lot of dirty corners".
+
+Now with the perspective of such awesome people out of the way let me explain why C++ sucks from my view(again I am not an expert).

```

Comment:
```suggestion
Now with the perspective of such awesome people out of the way let me explain why C++ sucks from my view (again I am not an expert).
```

Reply:
come on that's nit picking XD

Reply:
Nah bruh, that's Grammarly talking :laughing: